apps/web/pages/signup.tsx

```diff
@@ -247,319 +260,336 @@ export default function Signup({
-              <div className="mt-6">
-                <div className="relative flex items-center">
-                  <div className="border-subtle flex-grow border-t" />
-                  <span className="text-subtle leadning-none mx-2 flex-shrink text-sm font-normal ">
-                    {t("or_continue_with")}
-                  </span>
-                  <div className="border-subtle flex-grow border-t" />
-                </div>
-              </div>
-            )}
-            {/* Social Logins - Only for non-invite links*/}
-            {!token && (
-              <div className="mt-6 flex flex-col gap-2 md:flex-row">
-                {isGoogleLoginEnabled ? (
-                  <Button
-                    color="secondary"
-                    disabled={!!formMethods.formState.errors.username || premiumUsername}
-                    loading={isGoogleLoading}
-                    StartIcon={() => (
-                      <>
-                        <img
-                          className={classNames(
-                            "text-subtle  mr-2 h-4 w-4 dark:invert",
-                            premiumUsername && "opacity-50"
-                          )}
-                          src="/google-icon.svg"
-                          alt=""
-                        />
-                      </>
-                    )}
-                    className={classNames(
-                      "w-full justify-center rounded-md text-center",
-                      formMethods.formState.errors.username ? "opacity-50" : ""
-                    )}
-                    onClick={async () => {
-                      setIsGoogleLoading(true);
-                      const username = formMethods.getValues("username");
-                      const baseUrl = process.env.NEXT_PUBLIC_WEBAPP_URL;
-                      const GOOGLE_AUTH_URL = `${baseUrl}/auth/sso/google`;
-                      if (username) {
-                        // If username is present we save it in query params to check for premium
-                        const searchQueryParams = new URLSearchParams();
-                        searchQueryParams.set("username", username);
-                        localStorage.setItem("username", username);
-                        router.push(`${GOOGLE_AUTH_URL}?${searchQueryParams.toString()}`);
-                        return;
-                      }
-                      router.push(GOOGLE_AUTH_URL);
-                    }}>
-                    Google
-                  </Button>
+                {/* Password */}
+                <PasswordField
+                  data-testid="signup-passwordfield"
+                  label={t("password")}
+                  {...register("password")}
+                  hintErrors={["caplow", "min", "num"]}
+                />
+                {/* Cloudflare Turnstile Captcha */}
+                {CLOUDFLARE_SITE_ID ? (
+                  <TurnstileCaptcha
+                    appearance="interaction-only"
```

Comment:
If appearance is set to interaction-only, the widget will become only visible in cases where an interaction is required
 
 https://developers.cloudflare.com/turnstile/get-started/client-side-rendering/#appearance-modes

Reply:
Great find, that looks like what we want